src/operator/nn/activation.cc

```diff
@@ -182,6 +182,7 @@ The following activation functions are supported:
 })
 .set_attr<FCompute>("FCompute<cpu>", ActivationCompute<cpu>)
 #if MXNET_USE_MKLDNN == 1
+.set_attr<bool>("TIsMKLDNN", true)
```

Comment:
Is there a more generic way to add this instead of doing this for all operators? What if we later we add new operators, should we document this somewhere?

Reply:
if only needs to be added to MKLDNN operators. this fix was a temporary solution while we get the subgraph feature implemented. we weighed the pros / cons of waiting to release a stable MKLDNN stable with this hack or waiting another month for subgraph to be introduced (possibly with it's own bugs) and decided we would use this short term solution.

Reply:
Please mark this TODO and create a JIRA ticket to remove this later after MKLDNN support is released.

Reply:
added TODO to opexecuter